### DISCUSSION TITLE
Can I just get the current info at a station and NOT forecast information?

### DISCUSSION THREAD

**Discussion by JEDtoo** [weather-gov/api#259] [Q&A]
This seems like an obvious question but I couldn't find where anyone asked it.

I'd like the information like in a forecast but CURRENT readings with the time they were taken. Is there a way to request that?

Something like this:
 "generatedAt": "2020-12-12T20:44:25+00:00",
 "isDaytime": true,
 "temperature": 39,
 "temperatureUnit": "F",
 "temperatureTrend": null,
 "windSpeed": "9 mph",
 "windDirection": "N",
 "icon": "https://api.weather.gov/icons/land/day/rain,90?size=small",
 "shortWeather": "Light Rain",

**Comment by JJKraw** [comment]
Take a look at the /stations/{station ID}/observations/latest endpoint.

https://www.weather.gov/documentation/services-web-api#/default/get_stations__stationId__observations_latest

**Comment by JEDtoo** [comment]
Yes, that is exactly what I was looking for. Thank you!
